### DISCUSSION TITLE
OpenEHR Environment

### DISCUSSION THREAD

**Discussion by martysharkey** [microsoft/AzureTRE#4107] [Ideas]
As an Electronic Healthcare Record user i want to explore an opensource alternative using patient data within the secure confines of a TRE. 

**Is your feature request related to a problem? Please describe.**

This request is beginning to see the Azure TRE and associated workspaces as a secure marketplace which connects world class research and healthcare innovations into a secure data environment which provides protection for patient data. 

**Describe the solution you'd like**

I'd like to see the OpenEHR Opensource project and Java developed tools built to allow healthcare providers to test this. The OpenEHR project has the potential to disrupt global providers and the Azure TRE is a great environment to test, deploy and build this service longer term. 

**Describe alternatives you've considered**

There are many EHR providers globally and each provider has the ability to build their own workspace service in this open project irrespective of whether it is a commercial off the shelf product. This however places the EHR platform in the hands of an open community and the TRE offers a complementary deployment environment. 

**Additional context**

**Comment by Danny-Cooke-CK** [comment]
Thanks Martin, will add this to the roadmap

**Comment by marrobi** [comment]
@martysharkey  not sure I understand this request and how it fits into the Azure TRE roadmap. As far as I understand OpenEHR is a standard rather than a piece of software. 

Can you be more specific? Thanks.

**Comment by sconck** [comment]
I've moved this to a discussion pending any further input from @martysharkey or others on how OpenEHR fits
